### PR TITLE
Add Allies08b

### PR DESCRIPTION
**Pull request by Smittytron**
The scripts for Allies08a(#15582) and Allies08b are identical sans waypoints. There's a one line difference as Allies08a had one more beach to land on than Allies08b.

Using the timer conversion math from #15396 I had the mission timer at 27 min from the original, which had a 45 min timer. This seemed a bit long in testing, so I decided to shorten it to 21 min.

The player can also win early by destroying all Soviet forces on the map.

### CODE REVIEW

**Comment by abcdefg30** [comment]
Allies08a was merged.

**Comment by Smittytron** [comment]
Updated

**Review by abcdefg30 — DISMISSED** [review]
The reinforcements never move to attack the allied base but patrol their approach path instead.

File: mods/ra/maps/allies-08b/allies08b.lua [hunk 1]
```diff
@@ -0,0 +1,191 @@
...
+			c = HSLColor.White
+		end
+
+		Trigger.AfterDelay(DateTime.Seconds(i), function() UserInterface.SetMissionText("The experiment is a success!", c) end)
+	end
+	Trigger.AfterDelay(DateTime.Seconds(6), function() UserInterface.SetMissionText("") end)
+end
+
+DefendChronosphereCompleted = function()
+	local cells = Utils.ExpandFootprint({ ChronoshiftLocation.Location }, false)
```
Comment (abcdefg30) [inline mods/ra/maps/allies-08b/allies08b.lua]: `ChronoshiftLocation` does not exist.

**Comment by Muh-Muh** [comment]
The Evacuation objective didn't fire for me. The Scientists changed their owner as expected but just stood near the flare (where I ordered them to go earlier). It might have been caused by some mines I accidently placed at the start of the mission. Perhaps one was on the MCVEntry.location or blocked the path or something.

**Comment by abcdefg30** [comment]
Yes, they won't move if the path is blocked.

**Comment by Smittytron** [comment]
> The reinforcements never move to attack the allied base but patrol their approach path instead.

Fixed the other point, but haven't figured out the patrol thing yet. (It seems to work for everything but the south path) I'll keep trying to figure it out.

**Comment by Smittytron** [comment]
@abcdefg30 Moved the waypoint as suggested in IRC. Didn't see any units going backwards in my play-through.

**Comment by abcdefg30** [comment]
> Why is this not merged ?

I haven't retested the PR after the last set of changes.

**Review by MustaphaTR — APPROVED** [review]
Looks good to me.

**Review by obrakmann — APPROVED** [review]

**Comment by obrakmann** [comment]
[Changelog](https://github.com/OpenRA/OpenRA/wiki/Changelog-(bleed)/_compare/c606a75681df605eefd5d1e9c96957635fd727db)

**Pull request merged**